src/client.ts

```diff
@@ -1424,11 +1453,10 @@ export class MatrixClient extends EventEmitter {
         // We swallow errors because we need a default object anyhow
         return this.http.authedRequest(
             undefined, "GET", "/capabilities",
-        ).catch((e: Error) => {
+        ).catch((e: Error): {} => {
```

Comment:
What is the `: {}` doing...? Is that return type? But it doesn't return an object... 🤔